### PR TITLE
Optimization improvements

### PR DESCRIPTION
**Pull request by echarlaix**

### CODE REVIEW

File: examples/onnxruntime/optimization/token-classification/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+seqeval
+datasets >= 1.8.0
```
Comment (JingyaHuang) [inline examples/onnxruntime/optimization/token-classification/requirements.txt:2]: For `conll2003`, `datasets >= 1.18.0` is demanded.

File: examples/onnxruntime/optimization/token-classification/run_ner.py [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+from optimum.onnxruntime import ORTModel
+from optimum.onnxruntime.configuration import OptimizationConfig, ORTConfig
+from optimum.onnxruntime.optimization import ORTOptimizer
+
+
+# Will error if the minimal version of Transformers is not installed. Remove at your own risks.
+check_min_version("4.15.0")
+
+require_version(
+    "datasets>=1.8.0", "To fix: pip install -r examples/onnxruntime/optimization/token-classification/requirements.txt"
```
Comment (JingyaHuang) [inline examples/onnxruntime/optimization/token-classification/run_ner.py:47]: Same here, if we want `conll2003`, `datasets >= 1.18.0` is demanded.

File: examples/onnxruntime/optimization/question-answering/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Question answering
+
+The script [`run_qa.py`](https://github.com/huggingface/optimum/blob/main/examples/onnxruntime/optimization/question-answering/run_qa.py)
+allows us to apply dgraph optimizations using [ONNX Runtime](https://github.com/microsoft/onnxruntime) for question answering tasks.
```
Comment (JingyaHuang) [inline examples/onnxruntime/optimization/question-answering/README.md:20]:
```suggestion
allows us to apply graph optimizations using [ONNX Runtime](https://github.com/microsoft/onnxruntime) for question answering tasks.
```

**Review by JingyaHuang — APPROVED** [review]
Good job for the refactorization @echarlaix !

File: optimum/onnxruntime/configuration.py [hunk 4]
```diff
@@ -533,6 +533,58 @@ class OptimizationConfig:
     """
     optimize_with_onnxruntime_only: bool = False
 
+    """
+    Whether to disable the GELU fusion.
```
[the remark this answers is not in the document]
Comment (echarlaix) [reply optimum/onnxruntime/configuration.py:537]: Good catch, thanks !

File: examples/onnxruntime/optimization/token-classification/requirements.txt (hunk 1 above)
Comment (echarlaix) [reply examples/onnxruntime/optimization/token-classification/requirements.txt:2]: Good catch, thanks !

File: optimum/onnxruntime/configuration.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (echarlaix) [reply optimum/onnxruntime/configuration.py]: Not that I know of, this was actually added to stay consistent with the current docstring of `OptimizationConfig`. I plan to make some docstring cleaning in a coming PR.

**Pull request merged**